stubs/openpyxl/openpyxl/cell/cell.pyi

```diff
@@ -52,11 +52,11 @@ class Cell(StyleableObject):
     def check_string(self, value: str): ...
     def check_error(self, value: object) -> str: ...
     @property
-    def value(self) -> str | float | datetime | None: ...
+    def value(self) -> str | float | date | time | timedelta | None: ...
     @value.setter
-    def value(self, value: str | float | datetime | None) -> None: ...
+    def value(self, value: str | float | date | time | timedelta | None) -> None: ...
     @property
-    def internal_value(self) -> str | float | datetime | None: ...
+    def internal_value(self) -> str | float | date | time | timedelta | None: ...
```

Comment:
According to @Avasam's annotations in #9511, these could also be `bytes` or `decimal.Decimal`

Reply:
Unless you can show this can be restricted further, it does seem like anything valid for the function `get_type` (line ~64) can be a valid cell value type.

Reply:
I've extracted a type alias and added all types (except numpy types).